### DISCUSSION TITLE
Remove `Game engine` subtitle from logos

### DISCUSSION THREAD

**Discussion by lalitshankarch** [godotengine/godot-proposals#10863] [General Discussions]
Currently this logo is present throughout the engine:

![image](https://github.com/user-attachments/assets/9f905ba0-5247-4a99-824b-693a1e9033e7)

I think the `Game engine` subtitle is kind of redundant, especially when the editor is already open.

**Comment by Mickeon** [comment]
🤷 It doesn't really hurt to have.
